jupyter_book/directive/toc.py

```diff
@@ -19,6 +20,8 @@ def __init__(self, rawsource="", *children, text="Run code", **attributes):
 class TableofContents(SphinxDirective):
     def run(self):
         """returns an array of nodes for the tableofcontents directive declaration"""
+        if isinstance(self.env.app.builder, builders.latex.LaTeXBuilder):
```

Comment:
since this is a short-term fix, please add a comment that says what this is fixing and when we'll change it back